components/community/Header.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+
+import { HeadingLevel, HeadingTypeStyle } from '@/types/typography/Heading';
+
+import Button from '../buttons/Button';
+import IconRocket from '../icons/Rocket';
+import Heading from '../typography/Heading';
+
+interface HeaderProps {
+  className?: string;
+};
+
+/**
+ * @description This component displays Header.
+ * @param {HeaderProps} props - The props for Header component.
+ * @param {string} props.className - Additional CSS classes for styling.
+ */
+export default function Header({
+  // eslint-disable-next-line no-unused-vars, unused-imports/no-unused-vars
+  className = ''
```

Comment:
You can include `className` prop to add className to the top-level div below.

Reply:
Done